src/itr_ast.ml

```diff
@@ -1,5 +1,81 @@
+module Imandra_ptime = struct
```

Comment:
Realizing now that I can make this even better with the usage of `Imandra-extract`! Will give that a try later.